scripts/query-NPPES-resources.sh

```diff
@@ -27,4 +27,16 @@ rm *FileHeader.csv
 mv endpoint_pfile*.csv endpoint_pfile.csv
 mv npidata_pfile*.csv npidata_pfile.csv
 rm temp.zip
+
+echo "Removing all entries from npidata_pfile that are not Entity Type 1 (Organization)..."
```

Comment:
We want to remove all entries that are Entity Type 1, since 1 is Individual and 2 is Organization, sorry for any miscommunication!

Reply:
Since we're removing the individuals up front, we can remove line 454 in `nppesquerier/populatenpiorgs.go`

Reply:
Oops yeah I just wrote 1 instead of 2 by accident, I will fix that comment to say it is removing all entries that are not Entity type 2! Also good catch, I removed that line!